cleanlab_pinecone_rag_workflow/Cleanlab_Pinecone_RAG.ipynb

```diff
@@ -0,0 +1,2222 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "075f77c1-3328-4265-b077-160d496eb048",
+   "metadata": {},
+   "source": [
+    "# How to build a reliable, curated, and accurate RAG system using Cleanlab and Pinecone"
+   ]
```

Comment:
Curious: how come there is no mention of document chunking?  I would've thought the steps are:

1. chunk documents and embed/ingest chunks into pinecone DB
2. run Studio on every chunk as a separate example (with the text for the chunk coming from a file exported out of pinecone DB)
3. show how to map these results back to pinecone DB (for instance, delete: PII, non-English, toxic, and extra exact duplicate copies from the DB).

Not sure there's any need to run studio on entire documents at all in this tutorial, WDYT?

Reply:
I guess you've framed this as a one-time cleanup of documents prior to entering them into the RAG DB. While that's a valid use-case, I imagine it's less common than wanting to cleanup the existing RAG DB.

Reply:
we will do metadata in pt 2 of this notebook

Reply:
make sure to have a bunch of bad document chunks in the dataset

Reply:
workflow is:

pinecone DB -> Cleanlab -> better version of pinecone DB